config/kubeconfig/kubeconfig--gardener--prow-combined.yaml

```diff
@@ -0,0 +1,89 @@
+    extensions:
+    - extension:
+        gardenClusterIdentity: sap-landscape-live
+        shootRef:
+          name: prow
+          namespace: garden-garden-ci
+      name: client.authentication.k8s.io/exec
+    server: https://api.prow.garden-ci.shoot.live.k8s-hana.ondemand.com
+  name: shoot--garden-ci--prow
+- cluster:
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM5ekNDQWQrZ0F3SUJBZ0lSQUtaNGt5a2VwNjhqQi93am5qZFcyYk13RFFZSktvWklodmNOQVFFTEJRQXcKRlRFVE1CRUdBMVVFQXhNS2EzVmlaWEp1WlhSbGN6QWVGdzB5TVRFeE1qWXhOVFE0TURSYUZ3MHpNVEV4TWpZeApOVFE0TURSYU1CVXhFekFSQmdOVkJBTVRDbXQxWW1WeWJtVjBaWE13Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBCkE0SUJEd0F3Z2dFS0FvSUJBUURYVVd2K09GVzNVdnhtN0Q2cVZ4clAyeHhQa3A1VFBGWE5YVjZ0SEVtVHJrdEcKT0txSTZZNmxySTVuZEt3SnpRS2NXeEQwaHVodTB3TUJEb1JQS1lGY05vRTBYUkFsS2tLMFE0QzlRTHZQUFR0dgo1Slh3Z08yNVJXVEtRUWw1YVV5enRKdlRidlYzRDA5ZkZKOGFOemxNZE5mQndEUnhIOWlLemg5cGVwalV6bDV3CmZGVnRjdUUzTUxMT1d3OTBaM0QzNzNLTEpkU0pOK2M2S2RucmtvcUZtLy9yWmVscG9KSTJWZDg4VVZhNFhreDIKS3pkSlR1aWNQTVpvZGFXa2lGYmdIL0ZlRmZQNmlld1BhRm9HTERvQ0xXMTliZ09wckFlcmVCdERSenhTZmNjKwpWMC9Sc0NLQUljTnQ5NWZ6MHQ4RjNkTjRacmRRYXhReS9TMXR1WjY1QWdNQkFBR2pRakJBTUE0R0ExVWREd0VCCi93UUVBd0lCcGpBUEJnTlZIUk1CQWY4RUJUQURBUUgvTUIwR0ExVWREZ1FXQkJST0g3Zkk4ZXhKUEQ2Nk5jajQKb3ZxWUpISGxaVEFOQmdrcWhraUc5dzBCQVFzRkFBT0NBUUVBUnNZZGxFTmRzL0Urcld0Y2dGa2t4c1V4bUxWYQpydVVybnFDY0wrejFyVUNrazRLR3B4OS9MV3UrR0RPTmw2Y3ZVV213dnlpVWNueDFWZlFyY2h2ZDV5VmFaTFlsCnpDVFVhcWVpMDB0MGE1UW9YUXp5QWRuMUNjQXZDUVhmZ2RTZm9hNkRSSGRBQlI0MjF1UG90dCsvL2o1NFFnQ0MKNVFtb3R1cWFENGN1ZkFHeGxmTHAzWms2MHlqS09PN1dXcEQyQTBZdy94bUF5QTR1akF4bkNrRGVHUkVNdCs5cwppZ0FBaDdNOG5hREY2dytKVXE0Y3lDVVBMRkFjQXc2eit0dDhlME5la2Y3MFMrbVY1elpQSEhBOU81aGlyTlAyCnhaWW9FTjdrSVg2dW81ZFNOV2lXQkkwZTZ0K1N3L05VMGl4VXhnWjhXRG5yeXV1NUR3MzJLbFRtTEE9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
+    extensions:
+    - extension:
+        gardenClusterIdentity: sap-landscape-live
+        shootRef:
+          name: prow-work
+          namespace: garden-garden-ci
+      name: client.authentication.k8s.io/exec
+    server: https://api.prow-work.garden-ci.shoot.live.k8s-hana.ondemand.com
+  name: shoot--garden-ci--prow-work
+contexts:
+- context:
+    cluster: garden-garden-ci
+    namespace: garden-garden-ci
+    user: oidc-login
+  name: garden-garden-ci
+- context:
+    cluster: shoot--garden-ci--prow-work
+    user: garden-garden-ci--prow-work
+  name: gardener-prow-build
+- context:
+    cluster: shoot--garden-ci--prow
+    user: garden-garden-ci--prow
+  name: gardener-prow-trusted
+current-context: gardener-prow-trusted
+kind: Config
+preferences: {}
+users:
+- name: garden-garden-ci--prow
```

Comment:
`garden-garden-ci--prow` and `garden-garden-ci--prow-work` are equal, you could use one user configuration for both clusters.